tests/RuleSet/RuleSetTest.php

```diff
@@ -0,0 +1,467 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\RuleSet;
+
+use PhpCsFixer\ConfigurationException\InvalidForEnvFixerConfigurationException;
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\Fixer\DeprecatedFixerInterface;
+use PhpCsFixer\Fixer\PhpUnit\PhpUnitTargetVersion;
+use PhpCsFixer\FixerConfiguration\DeprecatedFixerOptionInterface;
+use PhpCsFixer\FixerFactory;
+use PhpCsFixer\RuleSet\RuleSet;
+use PhpCsFixer\RuleSet\RuleSets;
+use PhpCsFixer\Tests\TestCase;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\RuleSet\RuleSet
+ */
+final class RuleSetTest extends TestCase
```

Comment:
do i see right we stopped using AccessibleObject and we can drop it from package.json ?

Reply:
good catch, I've to do some more checks but I think we are no longer using it directly in the project anymore

Reply:
just to ensure we are on the same page, @SpacePossum , if we can drop it, then:
- in-repo `AccessibleObject` needs to remain there as public class, to be dropped on 3.0 line (maybe already dropped)
- external dependency `AccessibleObject` could be dropped

Reply:
on 2.15, the target of the PR, there is no external dependency, the `AccessibleObject` class in 2.15 has been deprecated in 2.5, so I think there is nothing more to do here in this PR?